server/src/test/java/org/elasticsearch/bootstrap/BootstrapChecksTests.java

```diff
@@ -194,6 +214,8 @@ boolean isMemoryLocked() {
                 "initial heap size [" + initialHeapSize.get() + "] " + "not equal to maximum heap size [" + maxHeapSize.get() + "]"
             )
         );
+        assertThat(e.getMessage(), containsString("; for more information see " +
+            "[https://www.elastic.co/guide/en/elasticsearch/reference/"));
```

Comment:
nit: combine string constants (may need reformatting)

```suggestion
        assertThat(e.getMessage(), containsString("; for more information see [https://www.elastic.co/guide/en/elasticsearch/reference/"));
```

(same comment for the other assertions added in this test suite)